src/cdn_definitions/schema.yaml

```diff
@@ -258,6 +261,9 @@ properties:
   exclude_from_listings:
     $ref: "#/definitions/exclude_versions_from_listings"
 
+  populate_ubi_dot_repos:
+    $ref: "#/definitions/populate_ubi_dot_repos"
```

Comment:
Minor: it seems overkill to use a $ref here.  It has generally been used to represent some kind of custom type (e.g. a string with additional restrictions on length or pattern, or a list/dict with certain kinds of elements), but here it's just a pure boolean.

Reply:
Thanks for the comment. Done.